src/lib.rs

```diff
@@ -141,18 +141,18 @@ fn parse_bucket(s: &str, label: &str) -> Option<f64> {
 
 #[derive(Debug, PartialEq)]
 pub struct HistogramCount {
-    less_than: f64,
-    count: f64,
+    pub less_than: f64,
+    pub count: f64,
 }
 
 #[derive(Debug, PartialEq)]
 pub struct SummaryCount {
-    quantile: f64,
-    count: f64,
+    pub quantile: f64,
+    pub count: f64,
 }
 
 #[derive(Debug, Eq, PartialEq)]
-pub struct Labels(HashMap<String, String>);
+pub struct Labels(pub HashMap<String, String>);
```

Comment:
Thanks for the PR!

This is the only line I'm not really sure about - exposing the `HashMap` inside `Labels` feels like an implementation detail that maybe someone might want to change in future. If exposing the inner value as `pub` now it would make that difficult.

The `Deref` impl gives you access to a map containing the labels in a read-only way, is that not enough for your use case? I'm not hard against changing it since I don't think this library sees a lot of use, but just want to make sure that change is really needed.